docs/api/Unity.Netcode.Interest.IInterestKernel-1.md

```diff
@@ -0,0 +1,59 @@
+---  
+id: Unity.Netcode.Interest.IInterestKernel-1  
+title: Unity.Netcode.Interest.IInterestKernel-1  
+---
+
+<div class="markdown level0 summary">
+
+</div>
+
+<div class="markdown level0 conceptual">
+
+</div>
+
+##### **Namespace**: System.Dynamic.ExpandoObject
+
+##### **Assembly**: MLAPI.dll
```

Comment:
I'm not sure why these are generated like this.

Reply:
all im doing is running the generator against the code, could it be someone/something has changed in the code. We have not changed how things are generated. from what i can see looking at the 0.0.1 API that format was also present in some of the entries, which makes me think its a style thing  in some places?